src/ErrorLogger/styles.js

```diff
@@ -27,7 +27,7 @@ export const ErrorsHeader = styled.div`
     box-shadow: ${props => props.headerShadow || '0 -2px 20px 0 rgba(20,31,60,0.65)'};
     border-top: ${props => props.headerTop || ' 1px solid #50637F'};
 
-    color: ${props => (props.errors ? '#FF4242' : '#19C6C7')};
+    color: ${props => (props.errors ? '#FFA8A8' : '#19C6C7')};
```

Comment:
This change would make us WCAG AAA compliant for normal text, but this is bolded text so `#FF6161` would be sufficient.

Reply:
👍

Reply:
#FF6161 is giving 2.9:1 contrast ratio (https://webaim.org/resources/contrastchecker/?fcolor=FF6161&bcolor=364C77)

Reply:
Ah good point, didn't check the hover background. Leave this as you changed then.